spec/unit/puppetlabs_spec_helper/tasks/module_inspect_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+
+describe 'rake module:inspect', type: :task, unless: puppet_3_or_older? do
```

Comment:
Should there be a matching test that it should error if puppet3 or older?

Reply:
Also, a pattern we've adopted in modules land is to use `skip(...)` moreso than guards, that way we can see the tests are being skipped as opposed to missing entirely.

e.g. https://github.com/puppetlabs/puppetlabs-powershell/commit/62ef1a5b9fcccc2f9dcf56fa511bac513c6633e3

Reply:
Yeah, my personal preference is to use the example group conditionals because I don't like having known and expected failure cases shown up as pending in the results. To me pending tests are more for "I know this thing is broken and I'm going to come back and fix it up later".

Regardless, this is a style decision that we should discuss as a team and decide on so that we can be consistent across the projects we're maintaining